charts/opentelemetry-kube-stack/examples/cloud-demo/values.yaml

```diff
@@ -0,0 +1,42 @@
+clusterName: demo
+opentelemetry-operator:
+  enabled: true
+defaultCRConfig:
```

Comment:
Yes I know I need to document this values file and what we expect for the output 😓